test/dotnet-format.Tests/tests/MSBuild/MSBuildWorkspaceFinderTests.cs

```diff
@@ -2,63 +2,78 @@
 
 using Microsoft.CodeAnalysis.Tools.Tests.Utilities;
 using Microsoft.CodeAnalysis.Tools.Workspaces;
+using Microsoft.NET.TestFramework;
 
 namespace Microsoft.CodeAnalysis.Tools.Tests.MSBuild
 {
-    public class MSBuildWorkspaceFinderTests
+    public class MSBuildWorkspaceFinderTests : SdkTest
     {
+
+        public MSBuildWorkspaceFinderTests(ITestOutputHelper log) : base(log)
+        {
+        }
```

Comment:
nit: empty line after ctor
```suggestion
        public MSBuildWorkspaceFinderTests(ITestOutputHelper log) : base(log)
        {
        }

```